tools/torch_quant/torch_quant/graph.py

```diff
@@ -283,10 +292,13 @@ def quantizable_module_to_observed(ctx: GraphModContext) -> None:
         w_ob_path = f'{node.target}.w_ob'
         w_ob = ctx.get_or_create_module(w_ob_path, ctx.w_ob_ctr)
         bias_ob = None
-        if getattr(src, 'bias', None) is not None and ctx.bias_ob_ctr:
-            bias_ob_ctr = partial(ctx.bias_ob_ctr, w_ob, act_ob)
+        if getattr(src, 'bias', None) is not None and is_observe_bias:
+            # If we use the existed bias observer, ctx.bias_ob_ctr should be None
             bias_ob_path = f'{node.target}.bias_ob'
-            bias_ob = ctx.get_or_create_module(bias_ob_path, bias_ob_ctr)
+            bias_ob = ctx.get_existed_module(bias_ob_path)
```

Comment:
nitpicking: could be simplified to `bias_ob = ctx.modules.get(bias_ob_path)` ?

Reply:
done